src/level.cpp

```diff
@@ -0,0 +1,44 @@
+#include "level.h"
+
+Level::Level() : Image() {}
+
+Level::~Level() {}
+
+bool Level::initialize(Game *gamePtr, TextureManager *textureM) {
+	spriteData.scale = (float)SCALE;
```

Comment:
Does `this->setScale((float)SCALE)` work? Better to do it through a setter mtd then modifying the variable directly.